css/main.css

```diff
@@ -0,0 +1,33 @@
+nav {
+  display: flex;
+}
+
+ul {
+  width: 100%;
+  list-style-type: none;
+}
+
+li {
+  float: right;
+  margin-right: 10px;
+}
+
+p {
+  width: 60%;
+  margin-left: 10px;
+  min-width: 150px;
+}
+
+.container {
```

Comment:
I know exactly what your other two class names refer to. Container, though, doesn't strike me as meaningful. Could this be more specific? What does it contain?